### ISSUE TITLE
FindBugs Warnings - File RawDiskHandler.java

### ISSUE THREAD

**Issue by ljdelight** [ljdelight/RawDiskService#10]
```
RawDiskHandler.java:41, OS_OPEN_STREAM, Priority: Normal
OS: com.ljdelight.rawdisk.RawDiskHandler.readLBAPretty(String, long) may fail to close stream

The method creates an IO stream object, does not assign it to any fields, pass it to other methods that might close it, or return it, and does not appear to close the stream on all paths out of the method.  This may result in a file descriptor leak.  It is generally a good idea to use a finally block to ensure that streams are closed.
```

```
RawDiskHandler.java:41, DM_DEFAULT_ENCODING, Priority: High
Dm: Found reliance on default encoding in com.ljdelight.rawdisk.RawDiskHandler.readLBAPretty(String, long): new java.io.InputStreamReader(InputStream)

Found a call to a method which will perform a byte to String (or String to byte) conversion, and will assume that the default platform encoding is suitable. This will cause the application behaviour to vary between platforms. Use an alternative API and specify a charset name or Charset object explicitly.
```

**Comment by ljdelight** [comment]
That change resolved it and didn't open more issues. Success!

**Comment by ljdelight** [comment]
This was resolved with commit f09223c, and pull request #11.

**Comment by ljdelight** [comment]
I followed the github documentation and it's not correctly closing the issue as fixed. CLOSING this myself.
